CharacterCreation/01_03_Races_Tundari

```diff
@@ -0,0 +1,171 @@
+==============================
+Tundari
+==============================
+
+===== Society =====
+
+The Tundari are a race of obligate carnivores from the planet Tundaar. Physically, 
+they span a wide size range, from the Chi’mau, who can range from 3’6” to 5’ and 
+85-130 lbs to the Sang’mau, at 6’ to 7’ and up to 400 lbs, and every size in-between. 
+Like humans, body types persist in many forms. 
+A sleek coat of fur covers these people, ranging widely in color and pattern. It 
+tends to grow short and fine over the majority of the body, however, the face, head, 
+neck, and joints have been known to grow hair that is longer and coarser, having a 
+mane or scruff-like quality about it. 
+As many cultures mark themselves with tattoos, the Tundari also have their own method
+of skin-marking. Some outliers will dye their fur, but that is generally frowned upon 
+by much of Tundari society. The traditional method of marking is called Aranga, 
+though many ranchers will would recognize the method as very similar to the freeze-
+branding they mark livestock with. Aranga, however, is far more intricate that simple 
+freeze-branding.
```

Comment:
I love this! These cultural descriptions make it really easy to draw inspiration for unique, non-stereotyped NPCs to RP with. Big improvement on existing system!!!